times_reader/transforms.py

```diff
@@ -1780,15 +1783,35 @@ def has_no_wildcards(list):
                 result.append(replace(table, dataframe=df, tag=datatypes.Tag.fi_t))
             else:
                 # wildcard expansion will happen later
+                # Handle named columns in data area:
                 if table.tag == datatypes.Tag.tfm_ins_ts:
-                    # ~TFM_INS-TS: Regions should be specified in a column with header=Region and columns in data area are YEARS
+                    # ~TFM_INS-TS: columns in data area are YEARS
                     data_columns = [
                         colname
                         for colname in df.columns
                         if colname not in known_columns | {"region", "ts_filter"}
```

Comment:
The code here assumes all columns that are not `known_colums` or `{"region", "ts_filter"}` are years